day_6/exercises/burrito.rb

```diff
@@ -13,7 +13,22 @@ def initialize(protein, base, toppings)
   end
 end
 
+ def add_topping(toppings)
+   @toppings.insert(2, 'onions') # add onions to toppings
+ end
+
+ def remove_topping(toppings)
+   @toppings.delete_at(2) # removes "guacamole" from toppings array
+ end
+
+ def change_protein
+   @protein.insert("Chicken") #inserts Chicken for protein
```

Comment:
What data type is `@protein`? I think you're looking for a different way to reassign `@protein` here.

Reply:
@protein is an object/instance variable.